### PR TITLE
Clean up UDPClient.

### PR DESCRIPTION
**Pull request by pgavlin**

### CODE REVIEW

**Comment by pgavlin** [comment]
@davidsh, @stephentoub PTAL

**Comment by davidsh** [comment]
LGTM

**Comment by stephentoub** [comment]
LGTM

File: src/System.Net.Sockets/src/System/Net/Sockets/UDPClient.cs [hunk 1]
```diff
@@ -7,48 +7,27 @@
...
         private AddressFamily _family = AddressFamily.InterNetwork;
 
-        // bind to arbitrary IP+Port
-        /// <devdoc>
-        ///    <para>
-        ///       Initializes a new instance of the <see cref='System.Net.Sockets.UdpClient'/>class.
-        ///    </para>
-        /// </devdoc>
+        // Initializes a new instance of the System.Net.Sockets.UdpClientclass.
         public UdpClient() : this(AddressFamily.InterNetwork)
```
Comment (CIPop) [inline src/System.Net.Sockets/src/System/Net/Sockets/UDPClient.cs:23]: Nit: For TcpClient you've moved the `: this(...)` on the next line. Which one should we use?

File: src/System.Net.Sockets/src/System/Net/Sockets/UDPClient.cs [hunk 2]
```diff
@@ -7,48 +7,27 @@
...
-        ///    </para>
-        /// </devdoc>
-
+        // Initializes a new instance of the System.Net.Sockets.UdpClientclass.
         public UdpClient(AddressFamily family)
         {
-            //
-            // Validate the address family
-            //
+            // Validate the address family.
```
Comment (CIPop) [inline src/System.Net.Sockets/src/System/Net/Sockets/UDPClient.cs:30]: Nit: I would rather remove comments that are just explaining clear code.

File: src/System.Net.Sockets/src/System/Net/Sockets/UDPClient.cs [hunk 3]
```diff
@@ -7,48 +7,27 @@
...
         {
         }
 
-        /// <devdoc>
-        ///    <para>
-        ///       Initializes a new instance of the <see cref='System.Net.Sockets.UdpClient'/>class.
-        ///    </para>
-        /// </devdoc>
-
+        // Initializes a new instance of the System.Net.Sockets.UdpClientclass.
```
Comment (CIPop) [inline src/System.Net.Sockets/src/System/Net/Sockets/UDPClient.cs:27]: Nit: I would rather remove comments that are just explaining clear code.

File: src/System.Net.Sockets/src/System/Net/Sockets/UDPClient.cs [hunk 4]
```diff
@@ -59,38 +38,27 @@ public UdpClient(AddressFamily family)
...
-        /// We should obsolete this. This also breaks IPv6-only scenarios.
-        /// But fixing it has many complications that we have decided not
-        /// to fix it and instead obsolete it post-Orcas.
+        // Creates a new instance of the UdpClient class that communicates on the
+        // specified port number.
+        //
+        // NOTE: We should obsolete this. This also breaks IPv6-only scenarios.
+        // But fixing it has many complications that we have decided not
+        // to fix it and instead obsolete it later.
         public UdpClient(int port) : this(port, AddressFamily.InterNetwork)
```
Comment (CIPop) [inline src/System.Net.Sockets/src/System/Net/Sockets/UDPClient.cs:47]: @SidharthNabar @josguil: Is this considered for removal? The comments are pointing out that this ctor should be obsolete.

File: src/System.Net.Sockets/src/System/Net/Sockets/UDPClient.cs [hunk 5]
```diff
@@ -292,12 +230,10 @@ public void AllowNatTraversal(bool allowed)
         private bool _cleanedUp = false;
```
Comment (CIPop) [inline src/System.Net.Sockets/src/System/Net/Sockets/UDPClient.cs:230]: Nit: maybe we should move this up (next to the other private members)?

File: src/System.Net.Sockets/src/System/Net/Sockets/UDPClient.cs [hunk 6]
```diff
@@ -631,9 +520,7 @@ public int Send(byte[] dgram, int bytes, string hostname, int port)
             }
```
Comment (CIPop) [inline src/System.Net.Sockets/src/System/Net/Sockets/UDPClient.cs:520]: Nit: new line after }

File: src/System.Net.Sockets/src/System/Net/Sockets/UDPClient.cs [hunk 7]
```diff
@@ -681,22 +558,16 @@ public int Send(byte[] dgram, int bytes)
...
```
Comment (CIPop) [inline src/System.Net.Sockets/src/System/Net/Sockets/UDPClient.cs:558]: Nit: new line after }

**Comment by CIPop** [comment]
A few code style comments.
LGTM otherwise.

**Pull request merged**